CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ The main work (all changes without a GitHub username in brackets in the below li
     - Feature: Automatically handle basicInformation uniqueId Property as defined by specification if not set by the developer
 
 -   @matter/nodejs
+    - Breaking: The StorageBackendDisk class got removed including the "node-localstorage" dependency. Use StorageBackendDiskAsync instead.
```

Comment:
Can we rename StorageBackendDiskAsync->StorageBackendDisk and make StorageBackendDiskAsync a deprecated alias?

Reply:
Alright I adjust that